lib/gui/app/models/local-settings.js

```diff
@@ -16,32 +16,42 @@
 
 'use strict'
 
+const electron = require('electron')
 const Bluebird = require('bluebird')
-const _ = require('lodash')
 const fs = require('fs')
 const path = require('path')
-const os = require('os')
-const Storage = require('./storage')
 
 /**
- * @summary Local storage settings key
+ * @summary Number of spaces to indent JSON output with
+ * @type {Number}
  * @constant
- * @type {String}
  */
-const LOCAL_STORAGE_SETTINGS_KEY = 'etcher-settings'
-const settingsStorage = new Storage(LOCAL_STORAGE_SETTINGS_KEY)
+const JSON_INDENT = 2
 
 /**
- * @summary Local settings filename
+ * @summary Userdata directory path
+ * @description
+ * Defaults to the following:
+ * - `%APPDATA%/etcher` on Windows
+ * - `$XDG_CONFIG_HOME/etcher` or `~/.config/etcher` on Linux
+ * - `~/Library/Application Support/etcher` on macOS
+ * See https://electronjs.org/docs/api/app#appgetpathname
  * @constant
  * @type {String}
  */
-const RCFILE = '.etcher.json'
+const USER_DATA_DIR = electron.remote.app.getPath('userData')
+
+/**
+ * @summary Configuration file path
+ * @type {String}
+ * @constant
+ */
+const CONFIG_PATH = path.join(USER_DATA_DIR, 'config.json')
 
 /**
  * @summary Read a local .etcherrc file
```

Comment:
Perhaps this should be changed to `Read the local configuration file` to match the jsdoc on line 82?

Reply:
Done